backend/prisma/schema.prisma

```diff
@@ -8,23 +8,24 @@ datasource db {
 }
 
 model event {
-  id              String        @id @default(uuid()) @db.Uuid
```

Comment:
Please undo the unnecessary reformatting.

Reply:
Didn't even notice that the prisma formatter formatted on save... 😅